Makefile

```diff
@@ -68,6 +70,10 @@ generate-client:
 	rm -rf client && mv maistra.io/api/client . && rm -rf maistra.io
 	## Hack - Because we are using core, client-gen hardcodes it to /api
 	find client -name core_client.go -exec sed -i 's|config.APIPath = "/api"|config.APIPath = "/apis"|' {} \;
+	## Hack - Because we are not using +groupName=maistra.io in doc.go, the incorrect group is used in fake types
```

Comment:
Does this mean this hack would go away if we add `+groupName=maistra.io` in the right place? If so, do you want to add a commit that does it so we don't need this?

Reply:
Adding the groupName changes the generated interfaces, which would require changes to maistra/istio.  We should do that as part of another PR/issue, but I would consider that low priority at this point.